RELEASE.md

```diff
@@ -13,8 +13,14 @@
 
 ## Major features and improvements
 
+* Added option to `SparkDataSet` to specify a `schema` load argument that allows for supplying a user-defined schema as opposed to relying on the schema inference of Spark.
+
 ## Bug fixes and other changes
 
+## Thanks for supporting contributions
+
+[Laurens Vijnck](https://github.com/lvijnck)
+
```

Comment:
```suggestion
```
(Not that your contribution isn't appreciated... just we don't do this section any more 🙂 )